languages/ruby/exercises/concept/conditionals-ternary/.docs/instructions.md

```diff
@@ -0,0 +1,39 @@
+In this exercise you will rewrite `if/else` statements from a movie theater's website into ternary conditionals.
+
+## 1. Check if a moviegoer is entitled to the seniors' discount
+
+Rewrite the `Moviegoer#ticket_price` method to utilize the ternary operator.
```

Comment:
```suggestion
Tickets at the movie theatre are $15 dollars.  However, if a person is 60 years of age or older, the ticket prices is discounted to $10.  Write the `Moviegoer#ticket_price` method to return the price of the ticket based on the movie goers' age utilizing the ternary operator.
```

Reply:
I really like the suggestion. A similar description can be used for the other tasks, especially with the "utilizing the ternary operator" bit.